_data/students.yml

```diff
@@ -108,3 +108,10 @@
   facebook: https://www.facebook.com/saicharan.reddy.927
   bio: New to github!
 
+
+- name: Pradeep Gangwar
```

Comment:
i dont think  you need to edit 53 files , you are doing this wrong , please make a new branch and send a pr 

please syc with master also